### PR TITLE
Add support for more networks to sourcify source fetcher

### PR DESCRIPTION
**Pull request by haltman-at**
Sourcify added more networks, updating!

### CODE REVIEW

File: packages/source-fetcher/lib/sourcify.ts [hunk 1]
```diff
@@ -59,6 +59,8 @@ const SourcifyFetcher: FetcherConstructor = class SourcifyFetcher
     "avalanche",
     "fuji-avalanche",
     "wagmi-avalanche",
+    //sourcify does *not* support non-testnet dfk?
```
Comment (cds-amal) [inline packages/source-fetcher/lib/sourcify.ts:62]: I'm confused by this question in `sourcify.ts` that precedes something named, `testnet-dfk-avalanche`.

**Review by cds-amal — COMMENTED** [review]
Hey @haltman-at, I have a question about the `non-testnet dfk` comment, otherwise lgtm.

File: packages/source-fetcher/lib/sourcify.ts (hunk 1 above)
Comment (haltman-at) [reply packages/source-fetcher/lib/sourcify.ts:62]: Yeah so DFK and WAGMI are Avalanche "subnets"?  I don't know what that actually means.  But you'll notice (per my practice of always including the mainnet -- or in this case the mainsubnet? -- in `networks.ts` even if it's not supported yet) I put both `dfk-avalanche` and `testnet-dfk-avalanche` in `networks.ts`.  These are different networks with different chain IDs.  However Sourcify only supports the latter atm?  So yeah I dunno.

Comment (cds-amal) [reply packages/source-fetcher/lib/sourcify.ts:62]: Thanks @haltman-at

**Review by cds-amal — APPROVED** [review]

**Pull request merged**